lib/test_state.go

```diff
@@ -26,6 +27,44 @@ type TestPreInitState struct {
 	TracerProvider *trace.TracerProvider
 	Usage          *usage.Usage
 	SecretsManager *secretsource.Manager
+
+	// FIXME (@oleiade): is this the way?
```

Comment:
I'm not 100% sure either, but perhaps this should belong to `Runner` or `TestRunState`, as `TestPreInitState` sounds more the place for things that remain static right after initialization 🤔